vlib/v/checker/tests/struct_type_cast_err.out

```diff
@@ -0,0 +1,77 @@
+vlib/v/checker/tests/struct_type_cast_err.vv:5:10: error: cannot cast `struct` to `string`
+    3 | fn main() {
+    4 |     foo := Foo{}
+    5 |     _ := string(foo)
+      |          ~~~~~~~~~~~
+    6 |     _ := int(foo)
+    7 |     _ := u64(foo)
+vlib/v/checker/tests/struct_type_cast_err.vv:6:10: error: cannot cast `struct` to `int`
+    4 |     foo := Foo{}
+    5 |     _ := string(foo)
+    6 |     _ := int(foo)
+      |          ~~~~~~~~
+    7 |     _ := u64(foo)
+    8 |     _ := u32(foo)
+vlib/v/checker/tests/struct_type_cast_err.vv:7:10: error: cannot cast `struct` to `u64`
+    5 |     _ := string(foo)
+    6 |     _ := int(foo)
+    7 |     _ := u64(foo)
+      |          ~~~~~~~~
+    8 |     _ := u32(foo)
+    9 |     _ := u8(foo)
+vlib/v/checker/tests/struct_type_cast_err.vv:8:10: error: cannot cast `struct` to `u32`
+    6 |     _ := int(foo)
+    7 |     _ := u64(foo)
+    8 |     _ := u32(foo)
+      |          ~~~~~~~~
+    9 |     _ := u8(foo)
+   10 |     _ := rune(foo)
+vlib/v/checker/tests/struct_type_cast_err.vv:9:10: error: unknown function: u8
```

Comment:
why is the error different for some types?

Reply:
there is no u8, it is `byte`

Reply:
Makes sense.  One error is if both types are known (what you're casting from and casting to).  The other is for an completely unknown type.

Reply:
Looks like I forgot to commit this change 😅